examples/patterns/forms/forms.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Forms / Layout
+category: _patterns
+---
+
+<form class="p-form p-form--stacked u-float-left">
```

Comment:
We should choose if inline our stacked is default and make that just `p-form` and then create a single modifier class for the other.